src/Template/Element/sidebar.ctp

```diff
@@ -1,21 +1,16 @@
 <?php
-use Cake\ORM\TableRegistry;
 use Cake\Utility\Inflector;
```

Comment:
Cant we make it so this is unnecessary, and the view receives all the data it needs pre-inflected?